src/source-updater.js

```diff
@@ -331,24 +335,45 @@ export default class SourceUpdater extends videojs.EventTarget {
       // used for debugging
       this.audioError_ = e;
     };
-    this.started_ = false;
+    this.createdSourceBuffers_ = false;
+    this.initializedEme_ = false;
+  }
+
+  initializedEme() {
```

Comment:
maybe this isn't a big deal, but should the source updater be the one to worry about initialized eme? to me that seems like a no, but if its the only place we can do it in a shared way, I kinda understand.

Reply:
I thought about that for a bit, and the reason I put it here was because it technically should worry about it, since Chrome's source buffers care about order of operations, source updater, being the closest to the source buffers, needs to worry as well.